experiment_1/scripts/FlexMex1_10/FlexMex1_10_preprocessing.py

```diff
@@ -271,23 +139,34 @@ def create_solar_pv_profiles():
 
 def main():
     # update elements
-    update_shortage_file()
-    update_load_file()
-    update_wind_onshore()
-    update_wind_offshore()
-    update_solar_pv()
-    update_link_file()
+    update_shortage_file(data_preprocessed_path)
+    update_load_file(data_preprocessed_path, scalars)
+    update_wind_onshore(data_preprocessed_path, scalars)
+    update_wind_offshore(data_preprocessed_path, scalars)
+    update_solar_pv(data_preprocessed_path, scalars)
+    update_link_file(data_preprocessed_path, scalars)
 
     # create sequences
-    create_load_profiles()
-    create_wind_onshore_profiles()
-    create_wind_offshore_profiles()
-    create_solar_pv_profiles()
+    create_load_profiles(data_raw_path, data_preprocessed_path)
+    create_wind_onshore_profiles(data_raw_path, data_preprocessed_path)
+    create_wind_offshore_profiles(data_raw_path, data_preprocessed_path)
+    create_solar_pv_profiles(data_raw_path, data_preprocessed_path)
+
+    # For consistency reasons in the output files: also drop indices for non-processed csv files
```

Comment:
Is it necessary to drop the index column ('region')? I do not expect it to cause any errors in the computation, but maybe I am wrong.

Reply:
I ran it without dropping the column and oemof.tabular didn't complain. I'll take this as a 'passed'.

Reply:
Resolved by https://github.com/modex-flexmex/oemo-flex/pull/20/commits/f43872abf289d7ab91cbd8098adbc924d990a3be